### DISCUSSION TITLE
Problema formulario entrega 1

### DISCUSSION THREAD

**Discussion by C0SS10** [rramosp/ai4eng.v1#279] [00 - General]
Me encuentro realizando el proyecto de forma individual, pero en el formulario que se envió para recopilar la información de la primera entrega me dice que es obligatorio poner un nombre. En las especificaciones del proyecto hay un apartado donde se dice que el proyecto se puede realizar de forma individual ¿Qué hago?

**Comment by Felipe-RA** [comment]
Saludos!

Gracias por el aviso! Ahora debería estar corregido

Hasta pronto!
